src/main/java/com/icthh/xm/ms/mstemplate/config/MicroserviceSecurityConfiguration.java

```diff
@@ -0,0 +1,63 @@
+package com.icthh.xm.ms.mstemplate.config;
+
+import com.icthh.xm.commons.permission.access.XmPermissionEvaluator;
+import com.icthh.xm.commons.permission.constants.RoleConstant;
+import com.icthh.xm.commons.security.jwt.JWTConfigurer;
+import com.icthh.xm.commons.security.jwt.TokenProvider;
+import com.icthh.xm.commons.security.spring.config.UnauthorizedEntryPoint;
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.security.access.expression.method.DefaultMethodSecurityExpressionHandler;
+import org.springframework.security.access.expression.method.MethodSecurityExpressionHandler;
+import org.springframework.security.config.Customizer;
+import org.springframework.security.config.annotation.method.configuration.EnableMethodSecurity;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configurers.AbstractHttpConfigurer;
+import org.springframework.security.config.annotation.web.configurers.HeadersConfigurer;
+import org.springframework.security.config.http.SessionCreationPolicy;
+import org.springframework.security.web.SecurityFilterChain;
+
+@RequiredArgsConstructor
+@Configuration
+@EnableMethodSecurity(securedEnabled = true)
+public class MicroserviceSecurityConfiguration {
```

Comment:
1 - this configuration already present in commons, pls override just method that allow to add endpoints
2 - overriding it`s only for customization, if it in template, then it have to be in commons!

Reply:
see com.icthh.xm.commons.security.spring.config.SecurityConfiguration